app/src/main/java/org/sopt/dosopttemplate/presentation/auth/LoginActivity.kt

```diff
@@ -50,20 +51,27 @@ class LoginActivity : AppCompatActivity() {
             val inputId = binding.etSignupId.text.toString()
             val inputPw = binding.etSignupPw.text.toString()
 
-            if (signUpUser != null && signUpUser.userId == inputId && signUpUser.userPw == inputPw) {
-                showShortToast(getString(R.string.login_success))
+            loginViewModel.loginUser(inputId, inputPw, this)
 
-                // 자동 로그인
-                if (binding.cbLoginAutologin.isChecked) {
-                    UserSharedPreferences.setUser(this, signUpUser)
-                }
+            loginViewModel.loginResult.observe(
+                this,
+            ) { loginSuccessful ->
+                if (loginSuccessful) {
+                    // showShortToast("getString(R.string.login_success)")
```

Comment:
```suggestion
when(loginSuccessful){
  true -> { 로직 }
  false -> { 로직 }
}
```
현재 if문이 중첩적으로 작성되었는데, when문으로 바꿔보면 조금 더 가독성 있는 코드를 작성할 수 있을듯합니다!!

Reply:
오!!!!!! if 대신 when문이 훨씬 다루기 좋을 것 같네요 앞으로 애용할게요~